src/utilities.ts

```diff
@@ -52,10 +53,11 @@ export function getRequestInformation(
     resourceType?: string;
     id?: string;
     vid?: string;
-    exportType?: ExportType;
+    export?: ExportAuth;

```

Comment:
When we create import will we need a similar datafield? if so could we plan for the future here?

Reply:
If follow other comment below this could be 'bulkAuth?: ExportAuth | ImportAuth;'